website/docs/r/network.html.markdown

```diff
@@ -6,10 +6,12 @@ description: |-
   Provides a vCloud Director VDC Network. This can be used to create, modify, and delete internal networks for vApps to connect.
```

Comment:
Please change all occurrences of "VDC Network" to "Org VDC Network", so it's clear that the context is organisation VDC as opposed to provider VDC.

Reply:
done